src/components/home/elements/TransactionRecords.vue

```diff
@@ -103,26 +107,31 @@
         class="type-select"
         router-tag="div"
         no-caret
+        :disabled="changeShowDisable || changeTypeShowDisable"
         variant="light">
         <template
           slot="button-content">
           <div style="display:inline-block; margin-right: 10px;">
-            <img
-              src="@/assets/imgs/icons/wallet/ic_filter.svg">
+            <img v-if="!changeTypeShowDisable"
+                 src="@/assets/imgs/icons/wallet/ic_filter.svg">
+            <img height="16"
+                 width="16"
+                 v-if="changeTypeShowDisable"
+                 src="@/assets/imgs/icons/wallet/ic_wait.svg">
             <span class="m-1"> Type </span>
           </div>
           <img src="@/assets/imgs/icons/signup/ic_arrow_down.svg">
         </template>
         <b-dropdown-item
           class="selection"
-          v-for="type in showTypes"
+          v-for="(typeValue, type) in showTypes"
           :key="type"
-          @click="changeType(type)">{{ type }}</b-dropdown-item>
+          @click="changeType(type)"> Type </b-dropdown-item>
```

Comment:
Wrong display word here. This should be specific type name.
<img width="473" alt="Screenshot 2019-08-26 at 4 30 53 PM" src="https://user-images.githubusercontent.com/41148850/63677177-0b431d80-c81f-11e9-9b7d-15536ad63fe4.png">